ZipBuilder/Sources/ZipBuilder/main.swift

```diff
@@ -16,14 +16,6 @@
 
 import Foundation
 
-// Delete the cache directory, if it exists.
```

Comment:
Hmm, considering a Firebase zip build can be 1GB and Carthage another 1GB, should we at least add an option to wipe the cache beforehand? It'd be pretty easy to get out of control for disk space here.

Reply:
Removing the working temp directory by default and keeping only with -keepBuildArtifacts true